packages/server-wallet/src/engine/engine.ts

```diff
@@ -438,14 +438,14 @@ export class SingleThreadedEngine
    * @param args - Parameters to create the channel with.
    * @returns A promise that resolves to the channel output.
    */
-  async createChannel(args: CreateChannelParams): Promise<MultipleChannelOutput> {
+  async createChannel(args: CreateChannelParams): Promise<SingleChannelOutput> {
```

Comment:
nit: it'd have been helpful to reviewers to split up b24e50c4a538d640489b92935ac355cc67db1adc, which does two things:
1. changes the `Engine.createChannel` API
2. changes the `Engine.createChannels` API